L103-core-move-insecure-creds-declaration.md

```diff
@@ -19,17 +19,25 @@ However, the issue https://github.com/grpc/grpc/issues/31012 points out the fact
 
 ## Proposal
 
-Move function declarations for insecure credentials from `grpc/grpc_security.h` to a new header file `grpc/grpc_insecure_credentials.h`.
+Move function declarations for each credential type from `grpc/grpc_security.h` to its own header file. The following is the list of mapping.
+
+* google_default_credentials: grpc/credentials/google_default.h
```

Comment:
Let's call the subdirectory `channel_credentials`.  That way we can also add a `call_credentials` directory later.